source/administration/update_mattermost.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+##Configuration - please adapt it to your environment
+#path to mattermost directory
+mattermostdir=/opt/mattermost
+#path for backup
+backupdir=/opt
+#temporary path for download
+downloaddir=/tmp
+#Team or Enterprise edition? Set 1 for team edition
+teamedition=0
+#start with plugins? Set 1 for starting with plugins active
+plugins=0
+#########
+
+##set version number from argument
+version=$1
+
+#set variable for echo output
+if [ $teamedition -ne 0 ]; then
+        edition=Team
+        downloadlocation=https://releases.mattermost.com/$version/mattermost-team-$version-linux-amd64.tar.gz
+else
+        edition=Enterprise
+        downloadlocation=https://releases.mattermost.com/$version/mattermost-$version-linux-amd64.tar.gz
+fi
+
+#delete old backup folder
+if [ -d "$backupdir/mattermost-backup" ]; then
+        echo Deleting old mattermost backup
+        rm -rf $backupdir/mattermost-backup
+fi
+
+###Get the file
+echo Downloading Mattermost $edition $version
+wget $downloadlocation -P $downloaddir -q
```

Comment:
Should we download first before deleting the old backup so that in case the download fails, the old back up is still available?

Reply:
Makes sense. I will update accordingly